requirements.txt

```diff
@@ -0,0 +1,5 @@
+torchvision
+tensorboardx
+scipy
+pyyaml
+easydict
```

Comment:
Can you add specific versions of the dependencies to be installed in the requirements.txt?
Like: 

- torchvision==version.
- scipy>=version